charts/business-central/values.yaml

```diff
@@ -134,6 +136,7 @@ service:
     dev: 7049
     # service.ports.dl -- Port of the service to access the files like the AL VS code extension via web. Remove to disable access.
     dl: 8080
+  sshEnabled: ""
```

Comment:
thinking about it again we can just check if `.Values.service.ports.ssh` is set so you can remove this new field and comment the new property in line 126 so SSH is disabled by default.